src/internal/api/authenticator.go

```diff
@@ -136,35 +165,13 @@ func (a *authenticator) handleFailedRefreshToken(err error) error {
 		return fmt.Errorf("received unauthorized error, re-login is required. %w", err)
 	}
 
-	switch a.status { //nolint
-	case statusReconnecting:
-		if a.attempts == a.cfgSvc.GetBackoffMaxAttempts() {
-			if err := a.handleConnectionFailed(err); err != nil {
-				return errors.Wrap(err, "failed to handle failed connection when reconnecting")
-			}
-
-			return fmt.Errorf("failed delayed attempt to refresh token. Re-login required. %w", err)
-		}
-
-		go a.hookResetToReconnecting()
-		a.status = statusWaitingToReconnect
-
-		return fmt.Errorf("failed delayed attempt to refresh token. Trying again. %w", err)
-	case statusWorkingProperly:
-		a.status = statusWaitingToReconnect
-		go a.hookResetToReconnecting()
-
-		return fmt.Errorf("failed to refresh token. Suspending for %v seconds. %w", a.cfgSvc.GetBackoffLength(), err)
-	default:
-		return fmt.Errorf("invalid auth status when refreshing token: %d. Error: %w", a.status, err)
-	}
+	return err
```

Comment:
Please add some context here as well (like `failed to refresh the auth token, try again later` or something).